gitops/snapshot_predicate.go

```diff
@@ -23,3 +24,22 @@ func IntegrationSnapshotChangePredicate() predicate.Predicate {
 		},
 	}
 }
+
+// SnapshotTestAnnotationChangePredicate returns a predicate which filters out all objects except
+// pull_request snapshot annotation "test.appstudio.openshift.io/status" is changed for update events.
+func SnapshotTestAnnotationChangePredicate() predicate.Predicate {
```

Comment:
nit: since the condition was changed to only take into account Snapshots that were created for pull requests, I think it would be helpful to update the name of the function and the docstring above to reflect that.

Reply:
Thanks, I thought to change a function name, but could not think out a short function name.

Reply:
Updated.

Reply:
Forgot to add suggestion: `PRSnapshotTestAnnotationChangePredicate`.

Reply:
Thanks.Updated.

Reply:
Thanks, looks good!